### DISCUSSION TITLE
CAS logout

### DISCUSSION THREAD

**Discussion by ward-verduyn** [akunzai/GSS.Authentication.CAS#578] [Q&A] [answered]
Hi! I'm quite new to using CAS but it was implemented in a project from someone else I currently work on. I do love your Nuget package because it simplifies A LOT! Now, my external login site has two different URL's for login & logout. Login is on login.site.com/cas, while logout is done using login.site.com/user/logout. How can I make sure I log my user out using that URL in the background.

**Comment by akunzai** [comment, marked as answer]
@ward-verduyn take a look the [AspNetCoreSample](https://github.com/akunzai/GSS.Authentication.CAS/tree/main/samples/AspNetCoreSample)

IMHO, The simplest way is modify the [CookieAuthenticationOptions.LogoutPath](https://learn.microsoft.com/dotnet/api/microsoft.aspnetcore.builder.cookieauthenticationoptions.logoutpath) and related controller.
